client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -100,10 +100,6 @@ export default React.createClass( {
 	wrongNSMappedDomains() {
 		debug( 'Rendering wrongNSMappedDomains' );
 
-		if ( this.props.selectedSite && this.props.selectedSite.jetpack ) {
```

Comment:
Just to be clear - this is intentional. We don't need this check anymore, since we don't show the domain warnings for Jetpack sites.